sdk/analytics/src/test/java/com/klaviyo/analytics/networking/requests/KlaviyoApiRequestTest.kt

```diff
@@ -44,17 +49,25 @@ internal class KlaviyoApiRequestTest : BaseRequestTest() {
     @Before
     override fun setup() {
         super.setup()
-
         every { networkMonitorMock.isNetworkConnected() } returns true
         every { configMock.networkTimeout } returns 1
         every { configMock.networkFlushIntervals } returns longArrayOf(10_000L, 30_000L, 60_000L)
     }
 
+    @After
```

Comment:
for clarity, I annotated all setup/cleanup, even though overrides technically inherit it